bids-validator/validators/tsv/checkTypeCol.js

```diff
@@ -0,0 +1,69 @@
+const Issue = require('../../utils').issues.Issue
+
+// allowable 'type' values from the BIDS specification
+const allowedMEEGTypes = ['EEG', 'ECOG', 'SEEG', 'DBS', 'PD', 'OTHER',  // (i)EEG
+    'EOG', 'ECG', 'EMG', 'EYEGAZE', 'GSR', 'HEOG', 'MISC', 'PUPIL', 'RESP', 'TEMP', 'VEOG', // physio
+    'AUDIO', 'REF', 'SYSCLOCK', 'TRIG', 'ADC', 'DAC', // system
+    'MEGMAG', 'MEGGRADAXIAL', 'MEGGRADPLANAR', 'MEGREFMAG', 'MEGREFGRADAXIAL', 'MEGOTHER', 'HLU', 'FITERR' // MEG
```

Comment:
From what I see, this is the complete list:

>{'MEGGRADPLANAR', 'ECG', 'EOG', 'REF', 'GSR', 'EYEGAZE', 'MEGGRADAXIAL', 'PUPIL', 'ECOG', 'AUDIO', 'MEGREFMAG', 'MEGREFGRADPLANAR', 'TRIG', 'EMG', 'MEGREFGRADAXIAL', 'VEOG', 'SYSCLOCK', 'ADC', 'TEMP', 'RESP', 'EEG', 'HEOG', 'FITERR', 'OTHER', 'HLU', 'SEEG', 'MISC', 'PD', 'MEGOTHER', 'DBS', 'MEGMAG', 'DAC'}


So one is missing from your list


```suggestion
    'MEGMAG', 'MEGGRADAXIAL', 'MEGGRADPLANAR', 'MEGREFMAG', 'MEGREFGRADAXIAL', 'MEGREFGRADPLANAR', 'MEGOTHER', 'HLU', 'FITERR' // MEG
```